src-php/Navigation/Pagination/Pagination.php

```diff
@@ -0,0 +1,165 @@
+<?php declare(strict_types = 1);
+
+namespace BrandEmbassy\Components\Navigation\Pagination;
+
+use BrandEmbassy\Components\UiComponent;
+use function ceil;
+use function file_get_contents;
+use function in_array;
+use function sprintf;
+use function str_replace;
+
+final class Pagination implements UiComponent
+{
+    private const BLANK = '';
+    private const FIRST_PAGE = 1;
+    private const MAX_PAGES_TO_DISPLAY = 5;
+    private const FIRST_PAGE_SVGS = ['arrow-first', 'arrow-back'];
+    private const LAST_PAGE_SVGS = ['arrow-next', 'arrow-last'];
+    private const CURRENT_PAGE_CLASS = 'class="current" ';
+    private const DISABLED_CLASS = 'class="disableClick"';
+
+    /**
+     * @var string
+     */
+    private $pageRoutePath;
+
+    /**
+     * @var int
+     */
+    private $totalItemCount;
+
+    /**
+     * @var int
+     */
+    private $pageNumberRequested;
+
+    /**
+     * @var int
+     */
+    private $totalPageCount;
+
+
+    public function __construct(
+        string $pageRoutePath,
+        int $totalItemCount,
+        int $pageNumberRequested,
+        int $totalPageCount
+    ) {
+        $this->pageRoutePath = $pageRoutePath;
+        $this->totalItemCount = $totalItemCount;
+        $this->pageNumberRequested = $pageNumberRequested;
+        $this->totalPageCount = $totalPageCount;
+    }
+
+
+    public function render(): string
+    {
+        $prevPageNumber = ($this->pageNumberRequested - 1) <= 0 ? 1 : $this->pageNumberRequested - 1;
+        $nextPageNumber = ($this->pageNumberRequested + 1) >= $this->totalPageCount
+            ? $this->totalPageCount
+            : $this->pageNumberRequested + 1;
+
+        return '<hr><div class="main">'
+            . '<div class="displayCounts">' . $this->renderDisplayCounts() . '</div>'
+            . '<div class="center">'
+            . $this->renderSvgAnchorTag('arrow-first', 1)
+            . $this->renderSvgAnchorTag('arrow-back', $prevPageNumber)
+            . '<div class="pagination">' . $this->renderPageAnchorTags() . '</div>'
+            . $this->renderSvgAnchorTag('arrow-next', $nextPageNumber)
+            . $this->renderSvgAnchorTag('arrow-last', $this->totalPageCount)
+            . '</div></div>';
+    }
+
+
+    public static function renderStyles(): string
+    {
+        $styles = file_get_contents(__DIR__ . '/css/Pagination.css');
+
+        if ((bool)$styles) {
+            return (string)$styles;
+        }
```

Comment:
```suggestion
        if (!$styles) {
            return self::BLANK;
        }
```
I would switch the statement like that so you don't need to type cast to bool